### PR TITLE
Minor changes to SCRAMBLE goggles. Third time is the charm.

### PR DESCRIPTION
**Pull request by MACIEKBAKI**
## About the Pull Request

Changes the variant of scramble goggles made in RnD (from normal to experimental, which have a chance of being faulty)
Changes the description of scramble goggles to be more formal, in-line with the setting
Prevents examining SCP-096 when blindfolded.

## Why It's Good For The Game

Makes getting convenient 096 protection atleast slightly risky
Description more formal to be in-line with the setting
You no longer anger 096 whilst blindfolded when you examine them.

## Did you test it?

Yes.

## Changelog

:cl:
tweak: Research and Development now makes experimental SCRAMBLE goggles instead of the normal variant.
tweak: Scramble goggle description changed to be more in-line with the setting.
bugfix: Being blindfolded and examing SCP-096 will no longer enrage him.
/:cl:

### CODE REVIEW

**Comment by MACIEKBAKI** [comment]
while i say scramble goggles, more like scramble goggles and one 096 bug(?)fix

File: code/modules/SCP/SCPs/SCP-096.dm [hunk 1]
```diff
@@ -214,6 +214,17 @@
 
 /mob/living/simple_animal/hostile/scp096/proc/specialexamine(var/userguy) //Snowflaked.
 	if (istype(userguy, /mob/living/carbon))
```
Comment (ivanmixo) [inline code/modules/SCP/SCPs/SCP-096.dm:216]:
```suggestion
	if(!iscarbon(userguy))
		return
```

Might as well early return asap.

File: code/modules/SCP/SCPs/SCP-096.dm [hunk 2]
```diff
@@ -214,6 +214,17 @@
 
 /mob/living/simple_animal/hostile/scp096/proc/specialexamine(var/userguy) //Snowflaked.
```
Comment (ivanmixo) [inline code/modules/SCP/SCPs/SCP-096.dm:215]:
```suggestion
/mob/living/simple_animal/hostile/scp096/proc/specialexamine(mob/userguy) //Snowflaked.
```
Why was it a var anyways? Turn the `var/userguy` a few lines up into `mob/userguy` as well. And turn the `..()` into `return ..()`

File: code/modules/SCP/SCPs/SCP-096.dm [hunk 3]
```diff
@@ -214,6 +214,17 @@
...
 	if (istype(userguy, /mob/living/carbon))
+		// Do not let blind folks examine 096. Doesn't make sense.
+		if(ishuman(userguy))
+			var/mob/living/carbon/human/H = userguy
+			if(H.equipment_tint_total == TINT_BLIND)
+				return
+		// Do not let unconscious or dead people examine 096.
+		// Dead as in ghost in dead body, not as in ghost
+		var/mob/M = userguy // we are sure he's atleast a mob
```
Comment (ivanmixo) [inline code/modules/SCP/SCPs/SCP-096.dm:224]:
```suggestion
```
Don't need this anymore, then

**Review by ivanmixo — CHANGES_REQUESTED** [review]

**Comment by MACIEKBAKI** [comment]
> Special_examine was also seperated because calling the examine proc remotely over a picture just failed.

the special examine was a 1:1 copy of examine which was funny
I made it also call from inside examine and just handle the special examine action stuff

**Comment by ivanmixo** [comment]
Don't see why not just do it right then and there in the examine proc, but whatever, should be fine.

**Comment by MACIEKBAKI** [comment]
> Don't see why not just do it right then and there in the examine proc, but whatever, should be fine.

Due to, as Yawet mentioned, issues when calling through anomalous photos, I believe?
I suppose maybe modularity?

**Comment by ivanmixo** [comment]
Seems ait.

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**